test/parallel/test-timers-active-handles.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const common = require('../common');
+const cp = require('child_process');
+const assert = require('assert');
+
+// compile deasync
+const nodeGypDir = '../../../deps/npm/bin/node-gyp-bin/';
+cp.spawn(
+      nodeGypDir + (common.isWindows ? 'node-gyp.cmd' : 'node-gyp'), ['rebuild'], {
+      stdio: 'inherit',
+      cwd: __dirname + '/test-timers-active-handles'
+    })
+  .on('exit', function (err) {
+    if (err) {
+      if (err === 127) {
+        console.error(
+          'node-gyp not found! Please upgrade your install of npm!'
```

Comment:
This should follow how tests/addons are compiled/run, as this here is very brittle.

(And it needs to go into tests/addons)